src/containers/Tenant/Diagnostics/TopQueries/TopQueries.tsx

```diff
@@ -43,19 +43,41 @@ const MAX_QUERY_HEIGHT = 10;
 const COLUMNS: Column<KeyValueRow>[] = [
     {
         name: 'CPUTimeUs',
-        width: 140,
-        sortAccessor: (row) => Number(row['CPUTimeUs']),
+        sortAccessor: (row) => Number(row.CPUTimeUs),
+        align: DataTable.RIGHT,
     },
     {
         name: 'QueryText',
         width: 500,
         sortable: false,
-        render: ({value}) => <TruncatedQuery value={value} maxQueryHeight={MAX_QUERY_HEIGHT} />,
+        render: ({row}) => (
+            <div className={b('query')}>
+                <TruncatedQuery value={row.QueryText} maxQueryHeight={MAX_QUERY_HEIGHT} />
+            </div>
+        ),
+    },
+    {
+        name: 'EndTime',
+        render: ({row}) => formatDateTime(new Date(row.EndTime as string).getTime()),
+        align: DataTable.RIGHT,
+    },
+    {
+        name: 'ReadRows',
+        render: ({row}) => formatNumber(row.ReadRows),
+        sortAccessor: (row) => Number(row.ReadRows),
+        align: DataTable.RIGHT,
+    },
+    {
+        name: 'ReadBytes',
+        render: ({row}) => formatNumber(row.ReadBytes),
+        sortAccessor: (row) => Number(row.ReadBytes),
```

Comment:
It was mentioned because we once discussed adding pagination to such tables. However, no pagination was added yet and we still request all the data from the source tables, so it no use in re-requesting data on sort